### PR TITLE
feat: add score multiple workloads example

### PR DESCRIPTION
**Pull request by mateuszjenek**

### CODE REVIEW

**Review by TobiasBabin — COMMENTED** [review]
Great stuff! I suggest we add at least some basic info how to work with the files, pls. see my commit suggestion.

File: score/multiple-workloads/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Deploy multiple workloads within one deployment
```
Comment (TobiasBabin) [inline score/multiple-workloads/README.md:1]:
```suggestion
This example shows how to deploy multiple workloads within one deployment.

Doing so requires a separate Score file per workload ([`score.example-service.yaml`](score.example-service.yaml) and [`score.another-service.yaml`](score.another-service.yaml)). The example also uses a Score extension file ([`score.humanitec.yaml`](score.humanitec.yaml)) to apply some common settings to both workloads.

The file [`deploy-config.yaml`](deploy-config.yaml) wraps all those files into a deploy configuration which can be passed in to `humctl` using the `--deploy-config` flag. See the [CLI documentation](https://developer.humanitec.com/platform-orchestrator/reference/cli-references/#score-integration) for details.
```

Comment (mateuszjenek) [reply score/multiple-workloads/README.md]: Sounds nice! Applied!

**Review by TobiasBabin — APPROVED** [review]
LGTM - thanks so much for contributing this part, it has been sorely missed 🎉

**Pull request merged**